### PR TITLE
fix: Use queries instead of api requests for wizard

### PR DESCRIPTION
**Pull request by HazAT**
The wizard now uses queries instead of internal API request, which should make it faster and more reliable.

### CODE REVIEW

**Comment by HazAT** [comment]
https://sentry.io/sentry/sentry/issues/413569782/

File: src/sentry/web/frontend/setup_wizard.py [hunk 1]
```diff
@@ -27,25 +28,28 @@ def get(self, request, wizard_hash):
         if wizard_data is None:
             return self.redirect_to_org(request)
 
-        orgs = client.get(
-            reverse('sentry-api-0-organizations'), request=request)
+        orgs = Organization.objects.filter(
+            member_set__role__in=[x.id for x in roles.with_scope('org:read')],
+            member_set__user=request.user,
+            status=OrganizationStatus.VISIBLE,
+        ).all().order_by('-date_added')[:50]
```
Comment (mattrobenolt) [inline src/sentry/web/frontend/setup_wizard.py]: nit: You don't need `.all()` in here.

**Review by mattrobenolt — APPROVED** [review]
So this will be a minor improvement, but this is still lots of O(n) going on.

For each organization, we're doing O(n) queries to fetch the projects.

And for each project, we do a query to get keys.

We can likely just do all of this in 3 big queries.

One to fetch all the orgs, one to get all of the projects for all of those orgs.

One to get all of the keys for all of the projects.

**Comment by HazAT** [comment]
I will merge this now and issue a follow-up PR for query/performance improvements.
People constantly running into this issue so this should be merged.

**Pull request merged**